File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/Application.java
package com.github.bumblebee;

import com.github.bumblebee.polling.LongPollingService;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

import javax.annotation.PostConstruct;

@SpringBootApplication
public class Application {

    private static final Logger log = LoggerFactory.getLogger(Application.class);

    @Autowired
    private LongPollingService longPollingService;

    @PostConstruct
    public void start() {

        longPollingService.startPolling();
        log.info("Started polling service");
    }

    public static void main(String[] args) throws Exception {
        SpringApplication.run(Application.class, args);
    }
}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/Application.kt
package com.github.bumblebee

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
open class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}